### PR TITLE
Run the test in a kind cluster

### LINKED ISSUE #2: Compile the code into a binary and add README.md
Add the description of this project and the running instruction into README.md. 
Example: https://github.com/K8sbykeshed/k8s-service-validator

### PR DESCRIPTION
**Pull request by iXinqi**
close https://github.com/K8sbykeshed/op-readiness/issues/2

### CODE REVIEW

**Comment by jayunit100** [comment]
@knabben  i might just suggest @iXinqi  could push at will on this repo  ... after all , if she breaks stuff she gets to fix it :) :) :)

File: Dockerfile [hunk 1]
```diff
@@ -0,0 +1,6 @@
+FROM golang:1.17 as build
```
Comment (knabben) [inline Dockerfile:1]: do they have the `golang:1.18beta1-slim` version? if not , can go with 1.17

File: kind_run.sh [hunk 2]
```diff
@@ -0,0 +1,113 @@
...
+        imagePullPolicy: IfNotPresent
+        command: [ "sleep" ]
+        args: [ "infinity" ]
+EOF
+    kubectl apply -f ${ARTIFACTS}/op-readiness-deployment.yaml
+    
+    i=0
+    sleep 30
+    OpReadinessPodStatus=$(kubectl get pod | grep "op-readiness" | sed "running")
+    while [ $i -lt 30 ] && [[ "$OpReadinessPodStatus" != *"Running"* ]]
```
Comment (knabben) [inline kind_run.sh]: what about something like:
`kubectl wait --for=condition=Ready --timeout=30s -l name=op-readiness`

**Review by knabben — APPROVED** [review]

File: Dockerfile (hunk 1 above)
Comment (iXinqi) [reply Dockerfile:1]: `golang:1.18beta1-slim` doesn't exist, let's use 1.17 for now. I can update it when `golang:1.18beta1-slim` is available.

File: kind_run.sh (hunk 2 above)
Comment (iXinqi) [reply kind_run.sh]: Sure, updated the code!

**Pull request merged**